iot/api-client/generate_keys.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Works from Cygwin / Bash for Windows AFAIK.  For a native solution, I have used PuTTyGen in the past.